### ISSUE TITLE
Device dropdown missing accessible name

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4095]
## Area or Component
Facility form

## Quick Summary
The device dropdown selector is missing an accessible name.

## Accessibility Violation
Select element must have an accessible name: https://dequeuniversity.com/rules/axe/4.4/select-name?application=AxeChrome

## Action Requested
Give each device selector element an accessible name (an aria-label). If there are multiple selectors, each likely needs a unique aria name.

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1346058064&range=4:5

**Comment by emmastephenson** [comment]
Note: this could be because the whole device component in the facility form is mis-coded. It's a table, but really should be a form group sharing a label. As it stands the fields are "labelled" by the table headers, but this is not best practice and should be changed.
